src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -215,6 +215,23 @@ public function get_items( $request ) {
 	 * @return true|WP_Error True if the request has read access for the item, WP_Error object otherwise.
 	 */
 	public function get_item_permissions_check( $request ) {
+		$template = $this->get_block_template( $request['id'] );
+		if ( is_wp_error( $template ) ) {
+			return $template;
+		}
+
+		if ( ! $template->wp_id ) {
+			return $this->permissions_check( $request );
+		}
```

Comment:
Is this for people getting multiple/all templates? It should be `$post_type_object->cap->read_posts` in that case, otherwise I'll need additional context so I don't give you a bum steer.

Reply:
Not all templates are loaded via a post. Some of them are hardcoded. If no wp I'd is set, then fallback to current behaviour.